grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -366,6 +366,7 @@ class QueryService implements Status {
         final String query =
             ' select c.id id, c.consent_key consentKey, c.project_key linkedProjectKey, c.pii pii, c.compliance compliance, c.sharing_type sharingType , c.text_sharing_type textSharingType, ' +
                     ' c.text_compliance textCompliance, c.require_mta requireMta, c.sample_collection_id sampleCollectionId, ' +
+                    ' c.phi phi, c.pii_dt piiDt, c.text_store textStore, c.publicly_available publiclyAvailable, c.store store, c.external_availability externalAvailability, c.genomic_data genomicData, ' +
```

Comment:
One of the things I've noticed is a pattern to query for many specific fields instead of querying for everything. I think it makes more sense to select c.*, etc. instead of making these kinds of small changes over time. This is one something that I'm dealing with a lot in the legacy Consent queries and I would recommend moving away from this pattern. If we queried for all fields, then this service method would not need any changes and the changes would be limited to the returned object.

So, not a specific request for this PR, but generic feedback that will make development more natural and flexible over time.

Reply:
I agree with you. I just followed the pattern but it can be improved.